python/lsst/ts/phosim/telescope/TeleFacade.py

```diff
@@ -671,7 +671,7 @@ def writeOpdInstFile(
 
         # Have to add 90 degrees rotation to account for the inconsistency
         # between phosim geometry and that of obs_lsst.
-        rot = self.surveyParam["rotAngInDeg"] + 90.0
+        rot = self.surveyParam["rotAngInDeg"]
```

Comment:
Please remove the comments of "Have to add 90...", this is not true anymore, and thanks for this improvement!